File: info/src/main/kotlin/renovation/info/data/model/TodoModel.kt
/*
 * Created under not commercial project "Renovation"
 *
 * Copyright 2021-2023
 */

package renovation.info.data.model

data class TodoModel(
    val id: Int,
    val title: String,
    val completed: Boolean,

    var description: String? = null,
    var category: String? = null,
    var image: String? = null,
    var price: Double? = null,
    var rating: Rating? = null,

    val date: String,
)

File: ng-part/src/app/data/model/todo.model.ts
/*
 * Created under not commercial project "Renovation"
 *
 * Copyright 2021-2023
 */

import {Rating} from "./rating.model"

export interface Todo {
  id: number
  title: string
  completed: boolean

  description?: string | null,
  category?: string | null,
  image?: string | null,
  price?: number | null,
  rating?: Rating | null,

  date?: Date | null
}
